models/ed/R/write.configs.ed.R

```diff
@@ -447,7 +447,7 @@ write.config.jobsh.ED2 <- function(settings, run.id){
   } else {
     modeloutdir <- file.path(settings$run$host$scratchdir, settings$workflow$id, run.id)
     mkdirscratch <- paste("mkdir -p", modeloutdir)
-    copyscratch <- paste("rsync", "-a", paste0('"', file.path(modeloutdir, "*"), '"'), paste0('"', outdir, '"'))
+    copyscratch <- paste("rsync", "-a", paste0('"', file.path(modeloutdir, ""), '"'), paste0('"', file.path(outdir, ""), '"'))
```

Comment:
Was there supposed to be a star, i.e. * between the two quotes after modeloutdir? @robkooper